### PR TITLE
Add some logging to SQSStream

### PR DESCRIPTION
**Pull request by alexwlchan**
🎹 🌴

### CODE REVIEW

File: sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala [hunk 1]
```diff
@@ -43,6 +43,7 @@ class SQSStream[T] @Inject()(actorSystem: ActorSystem,
           () => readAndProcess(streamName, message, process))
       }
       .map { m =>
+        println(s"Deleting message ${m.getMessageId}")
```
Comment (alicefuzier) [inline sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala]: I don't think we should be using println. It gets ordered in a different way than logged strings and printed strings give you less information.
Is there a reason to not use sl4j logging here?

File: sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala [hunk 2]
```diff
@@ -51,16 +52,21 @@ class SQSStream[T] @Inject()(actorSystem: ActorSystem,
     streamName: String,
     message: Message,
     process: T => Future[Unit])(implicit decoderT: Decoder[T]) = {
+    println(s"Processing message ${message.getMessageId}")
```
Comment (alicefuzier) [inline sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala]: Same

**Comment by kenoir** [comment]
What @alicefuzier said.

File: sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala (hunk 1 above)
Comment (alexwlchan) [reply sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala]: I’d forgotten we had it! Will fix.

**Pull request merged**